Add

ednoo opened the pull request:
Desafio.

ednoo committed:
Add

Desafio.

web11-ateves commented:
@edmesquita, faltou a resposta para o desafio 1. Por favor, coloque os dois no mesmo pull request.

ednoo commented:
Fiz os dois no mesmo arquivo. O <title>Desafio 2</title> é que está errado.
A princípio eu iria fazer em arquivos separados, mas achei mais prático
juntar os dois desafios num único arquivo.

@edmesquita https://github.com/edmesquita, faltou a resposta para o
desafio 1. Por favor, coloque os dois no mesmo pull request.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
https://github.com/Webeleven/desafio-estagio/pull/1#issuecomment-203968297

ednoo commented:
Foi o que eu entendi do enunciado, mas se estiver errado posso refazer sem
problemas.

function get_min_max() {

$numbers = array(9, 3, 4, 5, 6, 7);

$min = min($numbers);

$max = max($numbers);

$arr = implode(",",$numbers);

echo "Números = {";

echo $arr;

echo "}";

echo "<var>";

echo "<br />";

echo "O menor valor é: " . $min;

echo "<br />";

echo "O maior valor é: " . $max;

echo "</var>";

}

get_min_max();

Em 31/03/2016 1:15 PM, "Edno Mesquita" ednomesquita@gmail.com escreveu:

> Fiz os dois no mesmo arquivo. O <title>Desafio 2</title> é que está
> errado. A princípio eu iria fazer em arquivos separados, mas achei mais
> prático juntar os dois desafios num único arquivo.
> 
> @edmesquita https://github.com/edmesquita, faltou a resposta para o
> desafio 1. Por favor, coloque os dois no mesmo pull request.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/Webeleven/desafio-estagio/pull/1#issuecomment-203968297

web11-ateves commented:
Edno, talvez não tenha ficado claro no enunciado do problema 1, mas o usuário precisa informar a quantidade de números e depois informar os números (não pode ser fixo dentro de um array)
Fico no aguardo da resposta atualizada.

ednoo committed:
Fix

As operações e o resultado estavam omitidos quando o resultado era igual
a "null".
